Core/include/Acts/Seeding/SeedfinderConfig.hpp

```diff
@@ -74,6 +74,12 @@ struct SeedfinderConfig {
   Acts::Vector2 beamPos{0 * Acts::UnitConstants::mm,
                         0 * Acts::UnitConstants::mm};
 
+  // sets of consecutive phi bins in the seed making step
+  int numPhiNeighbors;
+  // vector containing the map of z bins in the top and bottom layers
+  std::vector<std::pair<int, int>> zBinNeighborsTop;
+  std::vector<std::pair<int, int>> zBinNeighborsBottom;
+
```

Comment:
This doesn't need to be here, right? Is this used anywhere? Since this is not part of the Seedfinder itself but rather the grid and the Binfinder, I'd leave the config confined to there.

Reply:
Why do you want to have them in any configuration? For the python tests?